packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -246,59 +245,22 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
     }
   }
 
-  public async createNetwork(community: Community) {
-    let network: NetworkData
-    // For registrar service purposes, if community owner
-    // FIXME: Remove if obselete
-    let network2: NetworkData
+  public async createNetwork(communityId: string): Promise<NetworkInfo | undefined> {
+    let network: NetworkInfo
+
     try {
       network = await this.getNetwork()
-      network2 = await this.getNetwork()
     } catch (e) {
-      this.logger.error(`Creating network for community ${community.id} failed`, e)
+      this.logger.error(`Creating network for community ${communityId} failed`, e)
       emitError(this.serverIoProvider.io, {
         type: SocketActionTypes.CREATE_NETWORK,
         message: ErrorMessages.NETWORK_SETUP_FAILED,
-        community: community.id,
+        community: communityId,
       })
       return
     }
 
-    this.logger(`Sending network data for ${community.id}`)
-
-    // It might be nice to save the entire Community in LevelDB rather
-    // than specific pieces of information.
-
-    const psk = community.psk
-    if (psk) {
-      this.logger('Creating network: received Libp2p PSK')
-      if (!isPSKcodeValid(psk)) {
-        this.logger.error('Creating network: received Libp2p PSK is not valid')
-        emitError(this.serverIoProvider.io, {
-          type: SocketActionTypes.CREATE_NETWORK,
-          message: ErrorMessages.NETWORK_SETUP_FAILED,
-          community: community.id,
-        })
-        return
-      }
-      await this.localDbService.put(LocalDBKeys.PSK, psk)
-    }
-
-    const ownerOrbitDbIdentity = community.ownerOrbitDbIdentity
-    if (ownerOrbitDbIdentity) {
-      this.logger("Creating network: received owner's OrbitDB identity")
-      await this.localDbService.putOwnerOrbitDbIdentity(ownerOrbitDbIdentity)
-    }
-
-    const payload: ResponseCreateNetworkPayload = {
-      community: {
-        ...community,
-        privateKey: network2.hiddenService.privateKey,
-      },
-      network,
-    }
-
-    this.serverIoProvider.io.emit(SocketActionTypes.NETWORK_CREATED, payload)
```

Comment:
Simplify createNetwork so that it only creates a network. Variables like PSK and ownerOrbitDbIdentity can be saved when creating the community since that's also when they are used.